src/main/kotlin/no/nav/helse/App.kt

```diff
@@ -99,34 +90,22 @@ fun Application.pleiepengesoknadapi() {
     }
 
     val idTokenProvider = IdTokenProvider(cookieName = configuration.getCookieName())
-    val jwkProvider = JwkProviderBuilder(configuration.getJwksUrl().toURL())
-        .cached(10, 24, TimeUnit.HOURS)
-        .rateLimited(10, 1, TimeUnit.MINUTES)
-        .build()
+    val issuers = configuration.issuers()
 
     install(Authentication) {
-        jwt {
-            realm = appId
-            verifier(jwkProvider, configuration.getIssuer()) {
-                acceptNotBefore(10)
-                acceptIssuedAt(10)
-            }
-            authHeader { call ->
-                idTokenProvider
-                    .getIdToken(call)
-                    .medValidertLevel("Level4")
+        multipleJwtIssuers(
+            issuers = issuers,
+            extractHttpAuthHeader = {call ->
+                idTokenProvider.getIdToken(call)
                     .somHttpAuthHeader()
             }
-            validate { credentials ->
-                return@validate JWTPrincipal(credentials.payload)
-            }
-        }
+        )
     }
 
     install(StatusPages) {
```

Comment:
Her tror jeg du må legge til `AuthStatusPages()` fra dusseldorf-ktor-auth tror jeg.
Legg til en test på om du mangler token/feil level etc og se om du får 403/401 som forventet.
Det trenger clienten for å fungere med redirect til login osv

Reply:
Ah, se du la det inn i `IdTokenStatusPages`. Funker det og 👍 
Så mulig disse testene allerede finnes? Da er det jo greit